script.js

```diff
@@ -0,0 +1,25 @@
+let userName = prompt('What is your name?');
+let userSurname = prompt('What is your Surname?');
+let favoriteNumber = prompt('What is your favorite number?');
+
+console.log(`Hello ${userName} ${userSurname}! Your favorite number is ${favoriteNumber}`);
+
+let num1 = prompt('Enter first number?',0);
+let num2 = prompt('Enter second number?',0);
```

Comment:
Better to be safe and do:

```suggestion
let num2 = +prompt('Enter second number?',0);
```